lib/browser/guest-view-manager.ts

```diff
@@ -203,13 +220,10 @@ const attachGuest = function (event, embedderFrameId, elementInstanceId, guestIn
     webSecurity: !params.disablewebsecurity,
     enableBlinkFeatures: params.blinkfeatures,
     disableBlinkFeatures: params.disableblinkfeatures,
+    ...Boolean(params.preload) && { preloadUrl: params.preloadURL },
```

Comment:
This `Boolean` trick shouldn't be needed, can we restore the original `if`

Reply:
Going to also add `preloadURL` to `WebPreferences` in this case as it isn't on that typing right now.